client/src/pages/ParentPortal.js

```diff
@@ -1,7 +1,5 @@
 import React, { Component } from 'react';
-import PrimaryNavBar from './../components/Navbar';
-import StudentCard from './../components/StudentCard';
-import AnnouncementList from './../components/Announcements';
+import { PrimaryNavBar, StudentCard, AnnouncementList } from './../components';
```

Comment:
I think the import name here should be PrimaryNavbar or change the component use on line 36 to PrimaryNavBar.

It gives an error while rendering right now

Reply:
Wait it doesn't give me an issue when I render it

Reply:
Oh damn it. I meant to comment in the PaymentPage.js file. That page has a <PrimaryNavbar /> but import PrimaryNavBar